src/main/java/net/dv8tion/jda/internal/requests/WebSocketClient.java

```diff
@@ -1319,51 +1319,52 @@ public <T extends SocketHandler> T getHandler(String type)
+        handlers.put("GUILD_SYNC",                             new GuildSyncHandler(api));
+        handlers.put("GUILD_STICKERS_UPDATE",                  new GuildStickersUpdateHandler(api));
+        handlers.put("GUILD_UPDATE",                           new GuildUpdateHandler(api));
+        handlers.put("INTERACTION_CREATE",                     new InteractionCreateHandler(api));
+        handlers.put("INVITE_CREATE",                          new InviteCreateHandler(api));
+        handlers.put("INVITE_DELETE",                          new InviteDeleteHandler(api));
+        handlers.put("MESSAGE_CREATE",                         new MessageCreateHandler(api));
+        handlers.put("MESSAGE_DELETE",                         new MessageDeleteHandler(api));
+        handlers.put("MESSAGE_DELETE_BULK",                    new MessageBulkDeleteHandler(api));
+        handlers.put("MESSAGE_REACTION_ADD",                   new MessageReactionHandler(api, true));
+        handlers.put("MESSAGE_REACTION_REMOVE",                new MessageReactionHandler(api, false));
+        handlers.put("MESSAGE_REACTION_REMOVE_ALL",            new MessageReactionBulkRemoveHandler(api));
+        handlers.put("MESSAGE_REACTION_REMOVE_EMOJI",          new MessageReactionClearEmojiHandler(api));
+        handlers.put("MESSAGE_UPDATE",                         new MessageUpdateHandler(api));
+        handlers.put("READY",                                  new ReadyHandler(api));
+        handlers.put("STAGE_INSTANCE_CREATE",                  new StageInstanceCreateHandler(api));
+        handlers.put("STAGE_INSTANCE_DELETE",                  new StageInstanceDeleteHandler(api));
+        handlers.put("STAGE_INSTANCE_UPDATE",                  new StageInstanceUpdateHandler(api));
+        handlers.put("THREAD_CREATE",                          new ThreadCreateHandler(api));
+        handlers.put("THREAD_DELETE",                          new ThreadDeleteHandler(api));
+        handlers.put("THREAD_LIST_SYNC",                       new ThreadListSyncHandler(api));
+        handlers.put("THREAD_MEMBERS_UPDATE",                  new ThreadMembersUpdateHandler(api));
+        handlers.put("THREAD_MEMBER_UPDATE",                   new ThreadMemberUpdateHandler(api));
+        handlers.put("THREAD_UPDATE",                          new ThreadUpdateHandler(api));
+        handlers.put("USER_UPDATE",                            new UserUpdateHandler(api));
+        handlers.put("VOICE_SERVER_UPDATE",                    new VoiceServerUpdateHandler(api));
+        handlers.put("VOICE_STATE_UPDATE",                     new VoiceStateUpdateHandler(api));
+        handlers.put("PRESENCE_UPDATE",                        new PresenceUpdateHandler(api));
+        handlers.put("TYPING_START",                           new TypingStartHandler(api));
+        handlers.put("APPLICATION_COMMAND_PERMISSIONS_UPDATE", new ApplicationCommandPermissionsUpdateHandler(api));
```

Comment:
Sort alphabetically.
Yes I know Presence and typing aren't sorted. They should be.